modules/core/arrow-test/src/main/kotlin/arrow/test/laws/HashLaws.kt

```diff
@@ -0,0 +1,29 @@
+package arrow.test.laws
+
+import arrow.typeclasses.Eq
+import arrow.typeclasses.Hash
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object HashLaws {
+  fun <F> laws(HF: Hash<F>, EQ: Eq<F>, cf: (Int) -> F): List<Law> =
+    listOf(
```

Comment:
Because Hash inherits from Eq then this implies that:

EqLaws.laws(...) +  listOf(...

This way you can test both Eq and Hash instances in a single call, replacing the current EqLaws call for HashLaws in the existing tests.